### DISCUSSION TITLE
🆘 nextjs i18n with defaultLanguage causes unwanted redirect

### DISCUSSION THREAD

**Discussion by Nexum** [vercel/community#408] [Help]
Hi,

I'm facing a big problem when using i18n vercel for some reason redirects all hits to / to /de/ ("de" being the default language), this is independent of user preference or headers. 

this does NOT happen if i build the app locally and just start with nextjs :(

Can anyone enlighten me how to stop vercel from doing this ? All my homepages are now /de/ instead of / :( 

Cheers
Nexum

**Comment by swarnava** [comment]
Hey there,

hank you for getting in touch! I'm sorry to hear you're facing this issue.

To help us debug more effectively, are you able to provide the following info/answers →

- A copy of your next.config.js
- Deployment URLs where we can reproduce this behavior
- What version of Next.js are you using? Have you recently upgraded/downgraded?
